doc(release note): add links to doc pages

NathalieC opened the pull request:
add link to profiles in studio
add link to SAML

NathalieC committed:
doc(release note): add links to doc pages

add link to profiles in studio
add link to SAML

tnguyen1 commented:
retest this please

benjaminParisel committed:
Merge branch '7.6' into release-note-links

benjaminParisel commented on md/release-notes.md:
```diff
@@ -8,17 +8,17 @@ Simply Bonita.
+### [Industrialization of Living Applications in Bonita Studio: Profiles](profileCreation.md)
+Our goal is to provide you with as few deployment artifacts as possible when creating applications.  
+To achieve this, we need all application resources to be available in the Studio / UI Designer development suite.  
+After application descriptors, profiles are now available in Bonita Studio.  
```
I suggest to put link on word [profiles] and not on section title...what did you tkink about this ?

NathalieC replied:
ok for me

benjaminParisel commented on md/release-notes.md:
```diff
@@ -8,17 +8,17 @@ Simply Bonita.
 
-### SAML logout
+### [SAML logout](single-sign-on-with-saml.md)
 SAML logout has been implemented, so that a user logging out of Bonita is also loggued out of all applications managed by the compagny Single-Sign-On system.
```
I suggest to put link on word [saml] and not on section title...what did you tkink about this ?

NathalieC replied:
good

benjaminParisel reviewed (commented):
Just 2 small  comments for you

NathalieC committed:
doc(RN 7.6): change links location

move profiles link to word and not title
move SAML logout link to word and not title

NathalieC commented:
Made the changes. Ready for a new review.

benjaminParisel commented:
LGTM
